src/promptflow/sample_trace.json

```diff
@@ -0,0 +1,290 @@
+            "span_type": "promptflow.Function",
+            "function": "builtins.sum",
+            "inputs": "{\n  \"iterable\": [\n    1,\n    2,\n    3\n  ]\n}",
+            "tool_version": "tool_version",
+            "output": "6"
+        },
+        "events": [],
+        "links": [],
+        "resource": {
+            "attributes": {
+                "service.name": "promptflow"
+            },
+            "schema_url": ""
+        }
+    },
+    {
+        "name": "stream",
+        "context": {
+            "trace_id": "0xa0f1ba62b17f09c0cc6fb6a176ed6b77",
+            "span_id": "0x93196722099aee47",
+            "trace_state": "[]"
+        },
+        "kind": "SpanKind.INTERNAL",
+        "parent_id": "0x764a9f505d713128",
+        "start_time": "2024-01-22T14:26:57.342076Z",
+        "end_time": "2024-01-22T14:26:57.342212Z",
+        "status": {
+            "status_code": "OK"
+        },
+        "attributes": {
+            "framework": "promptflow",
+            "span_type": "promptflow.Function",
+            "function": "stream",
+            "inputs": "{}",
+            "tool_version": "tool_version",
+            "output": "<streamed data, to be read later>"
+        },
+        "events": [],
+        "links": [],
+        "resource": {
+            "attributes": {
+                "service.name": "promptflow"
+            },
+            "schema_url": ""
+        }
+    },
+    {
+        "name": "stream.streamed_data",
+        "context": {
+            "trace_id": "0xa0f1ba62b17f09c0cc6fb6a176ed6b77",
+            "span_id": "0xa1806cfa742b3063",
+            "trace_state": "[]"
+        },
+        "kind": "SpanKind.INTERNAL",
+        "parent_id": "0x764a9f505d713128",
+        "start_time": "2024-01-22T14:26:57.342452Z",
+        "end_time": "2024-01-22T14:26:58.845260Z",
+        "status": {
+            "status_code": "UNSET"
+        },
+        "attributes": {
+            "framework": "promptflow",
+            "span_type": "promptflow.Function",
+            "function": "stream",
+            "inputs": "{}",
+            "tool_version": "tool_version",
+            "output": "[\n  \"0\",\n  \"1\",\n  \"2\"\n]"
+        },
+        "events": [],
+        "links": [
+            {
+                "context": {
+                    "trace_id": "0xa0f1ba62b17f09c0cc6fb6a176ed6b77",
+                    "span_id": "0x93196722099aee47",
+                    "trace_state": "[]"
+                },
+                "attributes": {}
+            }
+        ],
+        "resource": {
+            "attributes": {
+                "service.name": "promptflow"
+            },
+            "schema_url": ""
+        }
+    },
+    {
+        "name": "my_flow",
+        "context": {
+            "trace_id": "0xa0f1ba62b17f09c0cc6fb6a176ed6b77",
+            "span_id": "0x764a9f505d713128",
+            "trace_state": "[]"
+        },
+        "kind": "SpanKind.INTERNAL",
+        "parent_id": null,
```

Comment:
will there only 1 trave with parent id "null" in this json?

Reply:
UX needs to pick the information to render the top level overview information above the traces list.
![image](https://github.com/microsoft/promptflow/assets/5934735/ae6763e7-af52-4b85-a62e-acc7efba8ab0)